django-stubs/contrib/postgres/forms/array.pyi

```diff
@@ -0,0 +1,41 @@
+from typing import Any, Optional
+
+from django import forms as forms
+
+class SimpleArrayField(forms.CharField):
+    default_error_messages: Any = ...
```

Comment:
If attribute is defined on the base class, no need to define it on every field.

Reply:
addressed in 55abc1e77464908da2d53dd30f27b704c16d6d3d